src/Aspire.Hosting.Docker/DockerComposePublisherOptions.cs

```diff
@@ -19,4 +19,9 @@ public sealed class DockerComposePublisherOptions : PublishingOptions
     /// The name of an existing network to be used.
     /// </summary>
     public string? ExistingNetworkName { get; set; }
+
+    /// <summary>
+    /// Indicates whether to build container images during the publishing process.
+    /// </summary>
+    public bool BuildImages { get; set; } = true;
```

Comment:
Should negate this bool here?

Reply:
Have it default false and explicit opt in to builds?

Reply:
I'm not sure we have a way to flow command line arguments from the publisher, so this needs to be configured in code (I believe).

@mitchdenny ?

Reply:
A clean win without polluting cli could be to handle extra publisher options from IConfiguration and the AppHosts AppSettings?
Then just utilise the configuration binder Get<T> extension to populate an instance of "Optional" config?

Reply:
To be fair, could just be manually assigned, without the MergeExtension - we're only talking about a handful of options for each publisher - The helm one has more that Compose, and will potentially require even more for extension

I guess another option would be a new ManifestExpression, and have config directly in the AppHost declaratively

Reply:
We didn't get to mapping switches to config in 9.2 but as you've alluded the plan is to make use of IConfiguration. When registering publishers I'll probably have an optional mapping mechanism from switches to config sections.